apps/common/lib/lexical/math.ex

```diff
@@ -2,4 +2,11 @@ defmodule Lexical.Math do
   def clamp(number, min_number, max_number) do
     number |> max(min_number) |> min(max_number)
   end
+
+  def count_leading_spaces(str), do: count_leading_spaces(str, 0)
```

Comment:
This function doesn't seem like it belongs in the `Math` module, as it's more of a String thing.

Reply:
In your opinion, which file in the 'common' folder do you think is more suitable for placing this `count_leading_spaces` function?

Reply:
Ok, I moved to the `Lexical.Text`